test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -186,12 +186,27 @@ private void Init(WorkspaceModel workspace, EngineController controller, bool ds
                         }
                         else
                         {
-                            // TODO: Add support for multi-outport nodes.
-                           //foreach(var p in n.OutPorts)
-                           {
-                                var objs = n.GetCSValue(/*p.Index*/0, controller).Cast<object>();
-                                portvalues.AddRange(objs.Select(x => x.ToString()));
-                           };
+                            portvalues.AddRange(n.OutPorts.Select(p =>
+                            {
+                                var objs = n.GetCSValue(p.Index, controller).Cast<object>();
```

Comment:
what does `CS`? value mean?

Reply:
CSharp for want of a better name, I'm open to changing it.

Reply:
I couldn't use GetValue as it conflicts with an existing method. GetCLRValue maybe?